src/util/buildBlankReplacer.js

```diff
@@ -0,0 +1,6 @@
+export default counter => {
+  return () => `<span class='blank'>
```

Comment:
You could use [`dedent`](https://npm.im/dedent) here :bulb:

Reply:
We could but I am not a fan of adding additional packages here unless truly necessary.